teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './Person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(name, age)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Kindly fix the initialize method to request `age`, `name` and `parents_permission`.

- [ ] Also the `#super` method arguments are missing the correct order.

  ```
  def initialize(age, specialization, parents_permission: true, name: 'Unknown')
    super(age, parents_permission: parents_permission, name: name)
    @specialization = specialization
  end
  ```

Reply:
Thank you for the valuable feedback! @TedLivist  :smile: